src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -301,6 +306,17 @@ CHIP_ERROR BLEManagerImpl::_Init()
     return err;
 }
 
+void BLEManagerImpl::OnSendIndicationTimeout(System::Layer * aLayer, void * appState)
+{
+    uint8_t connHandle = 1;
```

Comment:
seems to be a hard-coded constant. `constexpr uint8_t kConnectionHandle = 1` ?

And a comment explaining why this handle is the correct value.

Reply:
connHandle we kept 1 right now because wifi BLE doesn't have any connection handle they identify connection using the peer BLE mac address so we need to replace the connection handle with mac ID so currently we kept for time being add TODO for the future